.travis.yml

```diff
@@ -0,0 +1,14 @@
+sudo: required
+dist: trusty
```

Comment:
Cool, this will give us access to `nvm` which can then be used to satisfy the JS dependencies